fe/fe-core/src/main/java/com/starrocks/catalog/TabletStatMgr.java

```diff
@@ -74,14 +80,10 @@
 public class TabletStatMgr extends FrontendDaemon {
     private static final Logger LOG = LogManager.getLogger(TabletStatMgr.class);
 
-    // for lake table
-    private final Map<Long, Long> partitionToUpdatedVersion;
```

Comment:
After reboot data will be lost, all partitions will re-collect tablet stats, I changed to record and persist the update time of tablet stats in `LakeTablet` to reduce the tablet stat collection task triggered after reboot